### PR TITLE
Fix for multiline insert queries

### PR DESCRIPTION
**Pull request by errx**
When insert query contains multiple lines batchmode is not working.

To fix that I have added regexp flag to match  "." to "\n"

### CODE REVIEW

File: stmt_test.go [hunk 1]
```diff
@@ -194,6 +194,18 @@ func (s *stmtSuite) TestFixDoubleInterpolateInStmt() {
 	s.NoError(rows.Close())
 }
 
+func (s *stmtSuite) TestFixMultiLineInsert() {
```
Comment (bgaifullin) [inline stmt_test.go]: Please add a new testcase for test TestExecMulti. 
like this:
```go
queries = []string{"INSERT INTO data (i64) VALUES (?)", ""INSERT INTO data(\ni64\n) VALUES (\n?\n)"}
for q in queries:
   //.. test case logic
```

**Comment by bgaifullin** [comment]
Thanks for your PR, there is just one comment about test case
Also seems like new golint does not work with go 1.8, I need to fix CI before merge this PR

**Comment by errx** [comment]
I've added a testcase

File: stmt_test.go (hunk 1 above)
Comment (bgaifullin) [reply stmt_test.go]: this test can be removed, since TestExecMulti covers this case

Comment (errx) [reply stmt_test.go]: done

**Comment by bgaifullin** [comment]
LGTM, thank you for PR

**Pull request merged**